### DISCUSSION TITLE
AdguardHome Service Question

### DISCUSSION THREAD

**Discussion by jsarmiento45** [AdguardTeam/AdGuardHome#5135] [General]
Hello,

I installed AdguardHome on Ubuntu Linux using the curl link provided in the documentation. Everything is working fine but I cannot stop the service I get either command not found or no such directory found when trying to stop the service. What directory do I need to be in order to stop/restart the service?

**Comment by horstmartinelli** [comment]
I have the same question. I was try to edit config .YAML
In my case, it was installed on a orangePi, but I cant find it on default directory "opt/AdGuardHome"
I hope it was to "var/snap" by using WinSCP, but the command "AdGuardHome -s stop" doesnt work. 

How can I stop this service?
Please

**Reply by jsarmiento45** [reply]
Try this command it worked for me. $ sudo /opt/AdGuardHome/AdGuardHome -s stop

**Reply by horstmartinelli** [reply]
Thanks for try, but unfortunately I got: sudo: /opt/AdGuardHome/AdGuardHome: command not found

**Reply by andekande** [reply]
run `sudo install.sh -v -r` to reinstall and see log outpout. There should be a line like `AdGuard Home will be installed into /opt/AdGuardHome`

**Reply by horstmartinelli** [reply]
Sorry, doesn't work. I got "-bash: line 9: install.sh: command not found"
My AdguardHome is in "/var/snap/adguard-home/4836"
There I see "leases.db" and "AdGuardHome.yaml" config file. No more archives
Im on right way?

I can see this files by WinSCP.
There is another word that I can search by WinSCP for sure?

**Reply by andekande** [reply]
apparently you have to use snap to control the adguard service:
`sudo snap stop adguard-home`
There are many locations snap uses. one of which is also `/snap/bin/`

**Reply by horstmartinelli** [reply]
Thanks a lot. It work with "sudo snap stop adguard-home"
But even with this, I cant edit "AdGuardHome.yaml", because I got error code 3 = "Permission Denied"

So, I try to change it with "sudo chmod 755 /var/snap/adguard-home/4808/AdGuardHome.yaml". And it change in WinSCP, but I got the same permission denied.
As Greta will say: 'Who dare you'

**Reply by andekande** [reply]
Yeah, snap sandboxes applications and I fear you'll have trouble with modified files when updating. 
On the other hand all settings can also be changed on the user interface.

**Reply by horstmartinelli** [reply]
I agree, but some settings I cant change by interface. In .txt file is visible any options that I would like to change, but I cant. Any idea how can do it even with service disable and permission to read? Thank you

**Reply by andekande** [reply]
honestly what I'd do is backup the yaml, then uninstall via snap
instead of snap you may re-install via https://github.com/AdguardTeam/AdGuardHome#automated-install-linux-and-mac
